include/cvc5/cvc5_kind.h

```diff
@@ -10,23 +10,35 @@
  * directory for licensing information.
  * ****************************************************************************
  *
- * The term kinds of the cvc5 C++ API.
+ * The term kinds of the cvc5 C++ and C APIs.
  */
 
-#include <cvc5/cvc5_export.h>
+#if (!defined(CVC5_API_USE_C_ENUMS) && !defined(CVC5__API__CVC5_CPP_KIND_H)) \
+    || (defined(CVC5_API_USE_C_ENUMS) && !defined(CVC5__API__CVC5_C_KIND_H))
 
-#ifndef CVC5__API__CVC5_KIND_H
-#define CVC5__API__CVC5_KIND_H
+#ifdef CVC5_API_USE_C_ENUMS
+#undef ENUM
+#define ENUM(name) Cvc5##name
+#else
+#include <cvc5/cvc5_export.h>
 
 #include <cstdint>
 #include <ostream>
-
 namespace cvc5 {
+#undef ENUM
+#define ENUM(name) class name
+#define EVALUE(name) name
+#endif
 
 /* -------------------------------------------------------------------------- */
 /* Kind                                                                       */
 /* -------------------------------------------------------------------------- */
 
+#ifdef CVC5_API_USE_C_ENUMS
+#undef EVALUE
+#define EVALUE(name) CVC5_KIND_##name
+#endif
+
```

Comment:
Why is this here and not included in the first #ifdef CVC5_API_USE_C_ENUMS above?  Seems inconsistent with how EVALUE is defined in the #else case.

Reply:
It has to be redefined for each enum.

Reply:
Note that cvc5_kind.h is not touched in this PR.